library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,62 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+For any particular library dependency chain, the libraries involved need to
```

Comment:
This sentence is long and a tad hard to follow. Can we shorten and break it up?

Reply:
I will rewrite the paragraph.

Reply:
yes...